week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -15,7 +25,13 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if (num === num && num % 2 === 0 && num <= 100) {
```

Comment:
not sure if there is a typo num===num will always return true
but maybe you want to check the type of it? (as we want to remove non number